client/my-sites/plugins/plugins-list/use-fields.tsx

```diff
@@ -121,7 +121,11 @@ export function useFields(
 				},
 				enableHiding: false,
 				render: ( { item }: { item: Plugin } ) => {
-					if ( item.status?.includes( PLUGINS_STATUS.UPDATE ) && item?.update?.new_version ) {
+					if (
+						item.status?.includes( PLUGINS_STATUS.UPDATE ) &&
+						item?.update?.new_version &&
+						! isListView
```

Comment:
We want to always hide this, right?

Reply:
The button is only hidden in list view when we have limited space.